common/utils.js

```diff
@@ -529,6 +529,31 @@ const parseTrustedChains = (config, trustedChains) => {
     return trustedChains.length === 1 && trustedChains[0] === 'all' ? itsEdgeChains(config) : trustedChains;
 };
 
+function stellarAddressToBytes(address) {
+    return hexlify(Buffer.from(address, 'ascii'));
+}
+
+function encodeDestinationAddress(destinationChain, destinationAddress) {
```

Comment:
```suggestion
function encodeITSDestination(destinationChain, destinationAddress) {
```
this is specifically for ITS, not clear from the name. Maybe explain via a docstring?

Reply:
make sense, updated